apps/chat/src/components/Chatbar/ExportModal.tsx

```diff
@@ -12,6 +17,15 @@ interface Props {
 }
 export const ExportModal = ({ onExport, onClose, isOpen }: Props) => {
   const { t } = useTranslation(Translation.SideBar);
+
+  const screenState = useScreenState();
+
+  useEffect(() => {
+    if (screenState !== ScreenState.MOBILE) {
+      onClose();
+    }
+  }, [onClose, screenState]);
```

Comment:
I think this useEffect should be in `ConversationContextMenu.tsx` instead of here
```
  const screenState = useScreenState();

  useEffect(() => {
    if (screenState !== ScreenState.MOBILE) {
      setIsShowExportModal(false);
      setIsShowMoveToModal(false);
    }
  }, [screenState]);
```